src/date-time/format.js

```diff
@@ -1,11 +1,13 @@
 import formatTime from './format-time.js';
 import formatDate from './format-date.js';
+import formatFuzzyDate from './format-fuzzy-date.js';
 import localeProvider from '../locale-provider.js';
 
 export default function DateTimeFormat(locale, options) {
 	options = options || {};
 	this.options = options;
 	this.localeData = localeProvider(locale, options.locale);
+	this._fuzzyDateFormatter = new formatFuzzyDate(this.localeData, this.options.locale, this.options.timeZone);
```

Comment:
Could you use lowercase `timezone` so that it's consistent [with the existing stuff](https://github.com/Brightspace/intl/blob/master/src/date-time/format-time.js#L7)?

Reply:
will do!

Reply:
So interestingly, Intl.DateTime accepts only the timezone Identifier eg "America/Winnipeg".  Time formatting uses the abbreviation.

Do you have a preference on the nomenclature?  I was thinking `option.timezoneidentifier`

Reply:
Sounds good to me